grakn-test/src/test/java/ai/grakn/test/graql/reasoner/Tests.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Grakn - A Distributed Semantic Database
+ * Copyright (C) 2016  Grakn Labs Limited
+ *
+ * Grakn is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Grakn is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Grakn. If not, see <http://www.gnu.org/licenses/gpl.txt>.
+ */
+
+package ai.grakn.test.graql.reasoner;
+
+import ai.grakn.graql.MatchQuery;
+import ai.grakn.graql.QueryBuilder;
+import ai.grakn.graql.internal.reasoner.query.QueryAnswers;
+import ai.grakn.test.GraphContext;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.ClassRule;
+import org.junit.Test;
+
+
+import static ai.grakn.test.GraknTestEnv.usingTinker;
+import static org.junit.Assume.assumeTrue;
+
+public class Tests{
+
+    @ClassRule
+    public static final GraphContext testSet10 = GraphContext.preLoad("testSet10.gql");
+
+    @Before
+    public void onStartup() throws Exception {
+        assumeTrue(usingTinker());
+    }
+
+    @Test
+    public void testSet10() {
```

Comment:
Please either rename this test or java doc what this test is actually testing. If anyone makes a change and this test breaks it will be very difficult to know what to look into. 

I know it is testing a specific graql file but what is the functionality being tested for example?